build/Dockerfile.nginx

```diff
@@ -4,11 +4,15 @@ FROM nginx:1.25.3-alpine
 ARG NJS_DIR
 ARG NGINX_CONF_DIR
 
-RUN apk update && apk upgrade && apk add --no-cache libcap \
+RUN apk update && apk upgrade && apk add --no-cache libcap curl ca-certificates \
+    && printf "%s%s%s\n" "http://nginx.org/packages/mainline/alpine/v" `egrep -o '^[0-9]+\.[0-9]+' /etc/alpine-release` "/main" | tee -a /etc/apk/repositories \
+    && curl -o /tmp/nginx_signing.rsa.pub https://nginx.org/keys/nginx_signing.rsa.pub \
+    && mv /tmp/nginx_signing.rsa.pub /etc/apk/keys/ \
+    && apk add nginx-module-otel \
```

Comment:
Question: do we want to always include this module? The upside is we don't need to create a separate Docker image. The downside is a bump in size (roughly 30MB I think).

Reply:
I think it's okay that we just include it right now. If our image starts to become unwieldy in the future we can explore ways to optimize.